address-watch/src/agent.js

```diff
@@ -1,34 +1,38 @@
 const { Finding, FindingSeverity, FindingType } = require('forta-agent');
 
 // load config file
-const addressList = require('./address-watch.json');
+const config = require('../bot-config.json');
 
-// load configuration data from agent config file
-const { aaveEverestId: AAVE_EVEREST_ID } = require('../agent-config.json');
+// load configuration data from bot config file
+const {
+  developerAbbreviation: developerAbbrev,
+  protocolName,
+  protocolAbbrev,
+  contracts,
+} = config;
 
-// get list of addresses to watch
-const addresses = Object.keys(addressList);
+function createAlert(contractAddress, contractName, type, severity) {
+  return Finding.fromObject({
+    name: `${protocolName} Address Watch`,
+    description: `Address ${contractAddress} (${contractName}) was involved in a transaction`,
+    alertId: `${developerAbbrev}-${protocolAbbrev}-ADDRESS-WATCH`,
+    type: FindingType[type],
+    severity: FindingSeverity[severity],
+  });
+}
 
 async function handleTransaction(txEvent) {
   const findings = [];
-  const { from, hash } = txEvent.transaction;
+  const { from } = txEvent.transaction;
+
+  Object.keys(contracts).forEach((contractName) => {
+    const { address } = contracts[contractName];
+    const { type, severity } = contracts[contractName].watch;
```

Comment:
Doesn't need to be changed, but you could swap `entries` for `keys` because you are going to be using the values anyway.  Then you could combine the destructure assignments to retrieve the nested values at the same time.
```suggestion
  Object.entries(contracts).forEach(([contractName, values]) => {
    const {
      address,
      watch: {
        type,
        severity,
      },
    } = values;
```